apidoc/Annotation.yml

```diff
@@ -159,4 +159,9 @@ properties:
     type: Boolean
     default: true
     since: "3.2.0"
-    
+
+  - name: visible
```

Comment:
Same here: Can you put `since: "6.0.0"` in here before next Friday? Then I'll try to get it into 6.0.0